modules/migration-performance-metrics.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc.adoc
+
+[id="migration-performance-metrics_{context}"]
+= Performance metrics
+
+The Migration controller records a set of metrics and pulls it into on-cluster monitoring storage. You can query the metrics by using PromQL to diagnose migration performance issues. All metrics are reset when the Migration controller manager container restarts.
+
+[id="provided-metrics_{context}"]
+== Provided metrics
+
+=== cam_app_workload_migrations
+
+This metric is a count of `MigMigration` CRs over time. It is useful for viewing alongside the `mtc_client_request_count` and `mtc_client_request_elapsed` metrics to collate API request information with migration status changes. This metric is included in Telemetry.
+
+.cam_app_workload_migrations metric
+[%header,cols="3,3,3"]
+|===
+|Queryable label name |Sample label values |Label description
+
+|status
+|`running`, `idle`, `failed`, `completed`
+|Status of MigMigration CR
+
+|type
+|`stage`, `final`
+|Type of MigMigration CR
+|===
+
+=== mtc_client_request_count
+
+This metric is a cumulative count of Kubernetes API requests that the Migration controller issued. It is not included in Telemetry.
+
+.mtc_client_request_count metric
+[%header,cols="3,3,3"]
+|===
+|Queryable label name |Sample label values |Label description
+
+|cluster
+|`\https://migcluster-url:443`
+|Cluster request was issued against
+
+|component
+|`migplan`, `migcluster`
+|Sub-controller API that issued request
+
+|function
+|`(*ReconcileMigPlan).Reconcile`
+|Function request was issued from
+
+|kind
+|`SecretList`, `Deployment`
+|K8s kind request was issued for
+|===
+
+
+=== mtc_client_request_elapsed
+
+This metric is a cumulative latency (in milliseconds) of Kubernetes API requests that the Migration controller issued. It is not included in Telemetry.
+
+.mtc_client_request_elapsed metric
+[%header,cols="3,3,3"]
+|===
+|Queryable label name |Sample label values |Label description
+
+|cluster
+|`\https://migcluster-url:443`
+|Cluster request was issued against
+
+|component
+|`migplan`, `migcluster`
+|Sub-controller API that issued request
+
+|function
+|`(*ReconcileMigPlan).Reconcile`
+|Function request was issued from
+
+|kind
+|`SecretList`, `Deployment`
+|Kubernetes kind request was issued for
+|===
+
+[id="accessing-metrics-in-{product-title}-4-monitoring-UI_{context}"]
```

Comment:
This ID is what is breaking your build. You cannot use attributes in XML IDs. Replace with 'ocp' or something else.